runbotics/runbotics-desktop/src/actions/sharepoint/SharepointActionHandler.ts

```diff
@@ -57,11 +65,22 @@ export class SharepointExcelActionHandler extends StatelessActionHandler {
         super();
     }
 
-    async open(input: SharepointOpenActionInput): Promise<SharepointOpenActionOutput> {
+    async openFileFromSite(input: SharepointOpenFromSiteActionInput): Promise<SharepointOpenActionOutput> {
+        const cloudPath = CloudPath.Site;
         const token = await this.microsoftSession.getToken();
-        await this.microsoftService.getFileIdA41SP(token.token, input.fileName);
-        await this.microsoftService.getWorksheetId(token.token, input.worksheetName);
-        return await this.microsoftService.createSession(token.token, input.persistChanges);
+        const siteId = await this.microsoftService.getSiteIdBySearch(token.token, input.siteName);
+        await this.microsoftService.getDriveId(token.token, siteId, input.listName);
+        await this.microsoftService.getFileIdBySearch(token.token, cloudPath, input.fileName);
+        await this.microsoftService.getWorksheetId(token.token, cloudPath, input.worksheetName);
```

Comment:
fns starting with `get` are meant to return value, do we intentionally ignore return value or there is none?